dev/fattest.simplicity/src/componenttest/rules/repeater/FeatureReplacementAction.java

```diff
@@ -314,6 +316,20 @@ public void setup() throws Exception {
             else
                 ClientConfigurationFactory.toFile(configFile, clientConfig);
         }
+
+        // Make sure config update is pushed to the wlp version of the server & client
+        if (serverConfigs.size() > 0) {
```

Comment:
Need to also handle the `ALL_SERVERS` case.  Otherwise line 323 will attempt `LibertyServerFactory.getLibertyServer("ALL_SERVERS");` which will fail